apps/prairielearn/src/ee/lib/aiQuestionGeneration.ts

```diff
@@ -534,6 +641,13 @@ Keep in mind you are not just generating an example; you are generating an actua
     return;
   }
 
+  if (completion?.usage?.prompt_tokens) {
+    job.data['promptTokens'] = completion?.usage?.prompt_tokens;
+  }
+  if (completion?.usage?.completion_tokens) {
+    job.data['completionTokens'] = completion?.usage?.completion_tokens;
+  }
```

Comment:
Nit, `completion` should never be nullish here, and I'd be inclined to skip the check (if the data is undefined, it just won't be serialized). Ditto above.

```suggestion
  job.data['promptTokens'] = completion.usage?.prompt_tokens
  job.data['completionTokens'] = completion.usage?.completion_tokens;
```

Reply:
I removed the check here and above 👍